server/routes/analyticsRouter.ts

```diff
@@ -63,13 +63,15 @@ const protectedCharacteristicsChartContent: Record<string, CharacteristicsChartC
       'This chart will help you tell if there is any imbalance in incentive levels between prisoners with a disability recorded on NOMIS and those without. As the amount of missing data is high, the number of prisoners with unknown information is also provided.',
     labelColumn: 'Disability',
     graphGoogleAnalyticsCategory: 'Incentive level by disability',
+    wideLabel: true,
```

Comment:
i added a type to describe this object in the other PR so you'll now need to put `wideLabel?: boolean` in there now

Reply:
Yep yep, test failed after we rebased. I just sorted this out 👌